build.sbt

```diff
@@ -229,6 +229,10 @@ lazy val cli = project
   .settings(
     allSettings,
     packSettings,
+    libraryDependencies ++= Seq(
+      // NB: The Mirror has an undeclared dependency on the scalahost-nsc package.
+      scalahostNsc(scalaVersion.value)
```

Comment:
scalahost-nsc is a fatjar that's only meant to be used as a compiler plugin via `-Xplugin:/scalahost-nsc.jar`. The third argument to `Mirror.apply` is the file path to the fatjar, it can be automatically detected if it's already on the classpath. I think for now, this is OK, but eventually we may want force scalafix-cli users to pass in this file path explicitly to avoid bloating the scalafix-cli classpath.